coreengine/audiomanager.cpp

```diff
@@ -568,17 +568,18 @@ void AudioManager::SlotAddMusic(QString file, qint64 startPointMs, qint64 endPoi
 #ifdef AUDIOSUPPORT
     if (!m_noAudio)
     {
```

Comment:
PLease revert this it has no disadvantage for linux but adds options for windows.